inMemoryBootstrap.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+cd docs/private_tangle
+
+echo "starting IRI"
+./02_run_iri.sh &
+until [ `docker ps | grep iri | cut -f1 -d\ ` ]; do
```

Comment:
Container will be visible in `docker ps` right away, what you want to check though if it has been alive for at least a few seconds without exiting / failing.

Reply:
No exactly, it may take some time to pull the image and dependancies, in practice it does spend some time waiting there, but I agree ofc that there can be a better test for liveness than `sleep 20`, which was an intentional eye sore for you to fix ;)